# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1352049818 21600
#      Sun Nov 04 11:23:38 2012 -0600
# Node ID 8be22193532b329d2ad47ab4624ad92932dc053d
# Parent  3bbba2b09587bf85531b1c0bbb66f7a845a78ce7
Add object name for main tool bar so that Qt does not complain at exit (bug #37498)

* main-window.cc (main_window::construct) Set the main_tool_bar object name
  to "MainToolBar" after instantiation.

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -738,16 +738,17 @@ main_window::construct ()
     = help_menu->addAction (tr ("Visit Octave Forge"));
   help_menu->addSeparator ();
 
   QAction * about_octave_action
     = help_menu->addAction (tr ("About Octave"));
 
   // Toolbars
   QToolBar *main_tool_bar = addToolBar ("Main");
+  main_tool_bar->setObjectName ("MainToolBar");
   main_tool_bar->addAction (new_script_action);
   main_tool_bar->addAction (open_action);
   main_tool_bar->addSeparator ();
   main_tool_bar->addAction (cut_action);
   main_tool_bar->addAction (copy_action);
   main_tool_bar->addAction (paste_action);
   main_tool_bar->addAction (undo_action);
   main_tool_bar->addAction (redo_action);
